informal/putnam.json

```diff
@@ -5065,6 +5065,14 @@
             "number_theory"
         ]
     },
+    {
+        "problem_name": "putnam_2022_a5",
+        "informal_statement": "Alice and Bob play a game on a board consisting of one row of 2022 consecutive squares. They take turns placing tiles that cover two adjacent squares, with Alice going first. By rule, a tile must not cover a square that is already covered by another tile. The game ends when no tile can be placed according to this rule. Alice's goal is to maximize the number of uncovered squares when the game ends; Bob's goal is to minimize it. What is the greatest number of uncovered squares that Alice can ensure at the end of the game, no matter how Bob plays?",
+        "informal_solution": "290",
```

Comment:
Could you modify these `informal_solution` fields to have an imperative sentence as the value, as opposed to a declarative one. Generally the format would be `Show that the solution is _` or something similar. I will include some general guidelines on this in an upcoming PR.

Reply:
Done!